steely/plugins/b.py

```diff
@@ -0,0 +1,37 @@
+COMMAND = '.b'
+__author__ = ('byxor',)
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    last_message = lambda: bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
+    send = lambda message: bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
+
+    send(encode(last_message()))
+
+    
+REPLACEABLE = [character for character in 'PpBbGg']
+REPLACEMENT = '🅱️'
+    
+
+def encode(string):
+    for sequence in REPLACEABLE:
+        string = string.replace(sequence, REPLACEMENT)
+    return string
+
+
+def help():
+
```

Comment:
This seems over engineered. Also, output will look a bit weird.
`B\nwill transform ...`

You could do a string format or something and I think it would be less obtuse

Reply:
I don't see the problem with it at the moment. The only reason it's obtuse is because we're writing help information as docstrings for the module.

Reply:
I got rid of the weird looking newline, which removed some of the logic from `help`. I think it'll make you happier.